### ISSUE TITLE
Unable to install R version of neurocombat onto Mac

### ISSUE THREAD

**Issue by tejaswidsudhakar** [Jfortin1/neuroCombat_Rpackage#5]
<img width="710" alt="Combat Harmonization Error" src="https://github.com/Jfortin1/neuroCombat_Rpackage/assets/101818403/06043a7b-cbd1-498f-8d9a-6d0a072e4af9">
<img width="877" alt="Screenshot 2023-05-31 at 11 43 20 PM" src="https://github.com/Jfortin1/neuroCombat_Rpackage/assets/101818403/78264cc4-19c4-4c23-8a60-9e70d29126d8">

Hi,

I tried installing this package several ways but it is not installing as a package in R. Can you help with this?

**Comment by emilyharriott** [comment]
I think that I am having a similar if not the same issue! I just posted "an issue" myself. https://github.com/Jfortin1/neuroCombat_Rpackage/issues/6

Did you manage to get it figured out?

Thank you so much!
Emily Harriott
PhD student, Vanderbilt University

**Comment by frank-fan-huang** [comment]
Me too, Could you help to take a look?

**Comment by Jfortin1** [comment]
@frank-fan-huang @emilyharriott @tejaswidsudhakar 

```
devtools::install_github("Jfortin1/neuroCombat_Rpackage")
devtools::install_github("Jfortin1/neuroCombatData")
```

**Comment by smucnyj13104** [comment]
hi, the commands still do not work, i am still getting the same 404 error. please help, thanks.

**Comment by emilyharriott** [comment]
@smucnyj13104 so sorry to hear it's not working! Below are some thoughts... 

I did a bunch of things to ultimately get mine to work -
install devtools package
update R
update R studio
update all packages
update BiocManager with BiocManager::install()
update BiocParallel with BiocManager::install("BiocParallel")

I also had to make sure my github was connected to my R/Rstudio (used chatgpt to help me figure out how to do that - still not sure if I did it the 'best' way but I had to install homebrew, download git, add git to my path in a zshrc file... (I'm using a mac)).

Then I ran this line

devtools::install_github("Jfortin1/neuroCombat_Rpackage")

and eventually I get this:

<img width="590" alt="Image" src="https://github.com/user-attachments/assets/f626643b-8dd5-4b23-ab26-c9a72deafe16" />


leading me to believe neurocombat was successfully installed! 



I'm not a programmer by any means, but maybe something there will help you? Let me know!? Best of luck!?
